### DISCUSSION TITLE
MILDEC: Dummy Units & Buildings

### DISCUSSION THREAD

**Discussion by Indy2222** [DigitalExtinction/Game#81] [Ideas]
See https://en.wikipedia.org/wiki/Dummy_tank

Make it possible to cheaply construct dummy units and building to deceive the enemy. The enemy would not be able to recognize them. These dummy units & buildings are easy to destroy and have no capabilities themselves.

**Comment by JackCrumpLeys** [comment]
I like the idea of making an army look much bigger than it truly is to destroy the moral of your opponent.
